test/sql/types/nested/map/map_from_entries/basic.test

```diff
@@ -0,0 +1,11 @@
+# name: test/sql/types/nested/map/map_from_entries/basic.test
```

Comment:
Can we rename this file to `data_types.test` or something? As far as I can tell so far all the test are either with integers, chars or nested types. I think we should add a few more test with strings, doubles, timestamps maybe. :)

Reply:
Definitely a good idea, I was putting it off because it's very tedious but worth doing, actually found missing functionality in MAP's key lookup when used together with INTERVAL